setup.cfg

```diff
@@ -16,7 +16,7 @@ install_requires =
     absl-py>=0.12.0
     flax==0.6.7
     jax==0.4.6
-    jaxlib>=0.4.6
+    jaxlib==0.4.6
```

Comment:
Also threw in this change because without it the test suite was installing jaxlib 0.4.7 and complaining that the jax version of 0.4.6 couldn't work with jaxlib 0.4.7.

Reply:
sounds good